### DISCUSSION TITLE
rustdesk has major issues after migration to new Mac

### DISCUSSION THREAD

**Discussion by ThomasWaldmann** [rustdesk/rustdesk#7384] [General]
### Bug Description

I used Apple's Migration Assistant to move all my stuff from my old Macbook Air M1 to my new Macbook Pro M3P.

After that, on the new Macbook:
- everything from "Recent Sessions" is gone (empty)
- Under "Your Desktop", under "ID", it says "Generating ..." - but never generates one. Under "One-time password", there is a random pw, but clicking on the "Refresh Password" icon, it does not generate a new random pw.
- Settings behave weird:
- Under "General" I can change the Theme, but the radiobutton dot does not move in the circle of the item I selected.
- Security: Nothing happens when I click on "Unlock Security Settings". And I can't change anything there as it stays locked.
- Network: same for "Unlock Network Settings"
- About: completely empty.

### How to Reproduce

Migrate from one to another Mac using Migration Assistant.

### Expected Behavior

Either it should work out of the box or it should at least offer some easy way to reset everything to defaults. I tried uninstalling and reinstalling rustdesk, but that did not help.

### Operating system(s)

macOS Sonoma (on both machines)

### RustDesk Version(s)

1.2.3 (on both machines)

**Comment by ThomasWaldmann** [comment]
I deleted the `com.carriez.RustDesk` folder from `~/Library/Preferences` and after that, rustdesk started working normally.

I still consider this as a usability bug though.

**Reply by antoinetielbeke** [reply]
Had the same issue, also would stop working and crashing for me all the time. Just migrated from an Intel mac to an M3. Thanks!

**Comment by oiasux** [comment]
Thanks, in my case this solved also the issues raised also at #376 and #3261
